.github/workflows/ci.yml

```
@@ -1,61 +1,83 @@
+        
+      - name: Upload build artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: dist
+          path: dist/
+
+  deploy:
+    needs: build
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
     
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)